client/executor/src/wasm_runtime.rs

```diff
@@ -283,6 +283,12 @@ pub fn create_wasm_runtime_with_code(
 	allow_missing_func_imports: bool,
 	cache_path: Option<&Path>,
 ) -> Result<Arc<dyn WasmModule>, WasmError> {
+	const WASM_CODE_BOMB_LIMIT: usize = 50 * 1024 * 1024;
+
+
```

Comment:
```suggestion
```

Please move the constant out of this function and add some docs.

Reply:
And why do we allow `100MB` in the wasm builder and only 50 mb here?

Reply:
Maybe we should put the constant into the `sp_maybe_compressed` crate to use every the same.

Reply:
I think it's better to have the constant closer to the usage. Initially I did have it in the crate, but then we started to talk about using the crate for PoVs as well. Those will want different max sizes.

Reply:
But we already got two constants with different values, in one pr.

Reply:
I'll add a `CODE_SIZE_BOMB_LIMIT` to the crate, but users will pass it through